auth/controllers/userController.ts

```diff
@@ -0,0 +1,79 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import User from "../models/user";
+import { addUserToDB, authUser } from "../services/AuthService";
+import { hashPassword, ReE, ReS, to } from "../services/Util";
+
+export async function loginUser(req: Request, res: Response): Promise<void> {
+  res.setHeader("Content-Type", "application/json");
+
+  const body = req.body;
+  let err;
+  let user: any;
+
+  [err, user] = await to(authUser(body));
+  if (err) {
+    return ReE(res, err, 422);
+  }
+  return ReS(res, { token: user.getJWT(), user: user.toJSON() }, 200);
+}
+
+export async function createUser(req: Request, res: Response): Promise<void> {
+  res.setHeader("Content-Type", "application/json");
+
+  const body = req.body;
+  let err;
+  let user: any;
+
+  if (!body.password) {  return ReE(res, "No password was entered", 422); }
```

Comment:
putting this on two lines would fit our existing coding standards more

Reply:
I may have been using an older linter config .. I will fix it and push